### ISSUE TITLE
pip install certbot-apache fails with pip 19

### ISSUE THREAD

**Issue by mgedmin** [certbot/certbot#6687]
If you have pip 19, `pip install certbot-apache` fails with

```
Exception:
Traceback (most recent call last):
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/cli/base_command.py", line 176, in main
    status = self.run(options, args)
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/commands/install.py", line 393, in run
    use_user_site=options.use_user_site,
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/req/__init__.py", line 57, in install_given_reqs
    **kwargs
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/req/req_install.py", line 919, in install
    use_user_site=use_user_site, pycompile=pycompile,
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/req/req_install.py", line 445, in move_wheel_files
    warn_script_location=warn_script_location,
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/wheel.py", line 583, in move_wheel_files
    outrows = get_csv_rows_for_installed(reader)
  File "/tmp/py3/lib/python3.6/site-packages/pip/_internal/wheel.py", line 565, in get_csv_rows_for_installed
    for fpath, digest, length in old_csv_rows:
ValueError: too many values to unpack (expected 3)
```

which is https://github.com/pypa/pip/issues/6165, but really caused by https://github.com/pypa/wheel/issues/280.

I've seen this ValueError in one of the comments in #6682, but certbot-auto is not the only affected use-case (although I understand that users who create virtualenvs by hand and pip install certbot-apace in there are not officially supported).

I'm filing this issue here because certbot-apache could avoid this issue if it renamed two test files

```
./certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old,default.conf
./certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-available/old,default.conf
```

to something that doesn't have a comma in the filename.

**Comment by adferrand** [comment]
@mgedmin, since pip 19.0.1 has been released is the issue still there after upgrade?

**Comment by ohemorange** [comment]
Keeping this thread up to date as well: yes.

**Comment by ohemorange** [comment]
Thanks for catching that @mgedmin, that worked like a charm! Implemented the fix in #6692.

**Comment by timdonovanuk** [comment]
So the temporary fix is to rename those 2 files? Cheers

**Comment by adferrand** [comment]
Version 0.30.1 is released, and contains the appropriate corrections to solve the issue with pip.

Installing certbot-apache through pip will work again.

**Comment by ohemorange** [comment]
Closed in #6692.
